### ISSUE TITLE
Improve the Selects

### ISSUE THREAD

**Issue by AerunDev** [PokemonWorkshop/PokemonStudio#113]
As a Developer
I would like to improve the Selects (accessibility & UX)
So that they are easier to maintain & respect the accessibility compliances

[See the technical wiki page about it](https://github.com/PokemonWorkshop/PokemonStudio/wiki/Select-as-input-tags)

## Acceptance criterias
- Make sure the Select follows the cursor when the user uses TOP/DOWN arrows

Things to check:
- [x] HTML popover
- [x] CSS anchor positioning

**Comment by NuriYuri** [comment]
Requirements:
1. Select shows their options regardless of if they're held into the main frame or a dialog frame
2. Select can be filtered (like ComboBox in Windows)
3. You can choose an option with up & down + enter
4. Pressing escape un-focus the select and closes its drawer
5. Pressing tab should lead to the next input option

Conceptually, the select can be a text input field with a portal'd or popover list. The input listen to up, down to communicate the index to the list, enter will choose the selected option based on index.

**Comment by NuriYuri** [comment]
I just checked HTML popover and CSS anchor.

HTML Popover works, it's not recognized by react as props but it's OK. One thing we have to do is being manual (the automatic stuff only works with buttons so we need to use the togglePopover() function).

CSS anchor is not working yet, the standard is still behind feature flags so there's probably no way to convince electron to make it work. Regardless, we can still set the position manually as I did in the example.

Here's few screenshots:
![image](https://github.com/PokemonWorkshop/PokemonStudio/assets/7335788/add46e72-0a58-462d-ac1d-a33632684e18)
![image](https://github.com/PokemonWorkshop/PokemonStudio/assets/7335788/0a7c07e0-c6a9-4d09-a22c-e974b23ceb91)

If you look at element inspector the popover is flagged as a top layer, like dialogs. This means it's not affected by overflow or scrolling:
![image](https://github.com/PokemonWorkshop/PokemonStudio/assets/7335788/b5bdb3b5-d113-4b70-9a55-2201ce12f4fc)

The bad new with that is that we might need handle scrolling of the parent element (one solution would be closing the popover).

Btw since popover are behaving a bit like dialogs, we can use them for the tooltip as well :)
